server/protocols/protocols-smtp/src/main/java/org/apache/james/smtpserver/SendMailHandler.java

```diff
@@ -77,8 +77,7 @@ public HookResult onMessage(SMTPSession session, Mail mail) {
                 recipientString = theRecipients.toString();
             }
             if (LOGGER.isInfoEnabled()) {
-                String infoBuffer = "Successfully spooled mail " + mail.getName() + " from " + mail.getSender() + " on " + session.getRemoteAddress().getAddress().toString() + " for " + recipientString;
-                LOGGER.info(infoBuffer.toString());
+                LOGGER.info("Successfully spooled mail " + mail.getName() + " from " + mail.getSender() + " on " + session.getRemoteAddress().getAddress().toString() + " for " + recipientString);
```

Comment:
You can remove the if and use placeholder here I think

Reply:
Done.  Additionally, I merged the`recipientString` calculation into the info() logger call.